test/box-luatest/gh_9823_net_replicaset_test.lua

```diff
@@ -322,7 +325,10 @@ g.test_watch_leader = function(cg)
     -- Change the leader.
```

Comment:
Found another flaky fail when testing locally:
```
[002] not ok 4T>box-luatest.gh_9823_net_replicaset.test_watch_leader                                                           
[002] #   ...e/sptnt/test/box-luatest/gh_9823_net_replicaset_test.lua:295: expected: 1, actual: 2
[002] #   stack traceback: 
```

Reply:
Added another fix.

Reply:
Thanks! I can't reproduce the failure anymore.